src/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.links {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 50px;
+}
+
+.logo {
+  padding-top: 10px;
+}
+
+.list {
+  display: flex;
+  align-items: stretch;
+
+  height: 60px;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_link {
+  margin: 0 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.nav_link:first-child {
+  margin-left: 0;
+}
+
+.nav_link_a {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav_link:last-child {
+  margin-right: 0;
+}
+
+.nav_link_a:hover {
+  color: #00acdc;
+  cursor: pointer;
+}
+
```

Comment:
You forgot to add pseudo-class `:visited` to your links

Reply:
How my links are supposed to change when they are visited?

Reply:
They must be blue(#00acdc)